website/pages/docs/post-processors/vagrant.mdx

```diff
@@ -113,6 +113,67 @@ more details about certain options in following sections.
   creation of the Vagrantfile at some previous point in the build.
   Defaults to `false`.
 
+## Using together with the Artifice post-processor
+
+Sometimes you may want to run several builds in a pipeline rather than running
+this post-processor inside a long-running Packer build. Here is an example of
+how to do this:
+
+<Tabs>
+<Tab heading="JSON">
+
+```json
+{
+  "builders": [{
+    "type": "null",
+    "communicator": "none"
+  }],
+  "post-processors": [
+    [
+      {
+        "type": "artifice",
+        "files": ["output-virtualbox-iso/vbox-example-disk001.vmdk",
+              "output-virtualbox-iso/vbox-example.ovf"
+          ]
+      },
+      {
+        "type": "vagrant",
+        "keep_input_artifact": true,
+        "provider_override": "virtualbox",
+      }
+    ]
+  ]
+}
+```
+
+</Tab>
+<Tab heading="HCL2">
+
+```hcl
+source "null" "example" {
+    communicator = "none"
+}
+
+build {
+  sources = [
+    "source.null.example"
+  ]
+
+  post-processor "artifice" {
+    files = ["output-virtualbox-iso/vbox-example-disk001.vmdk",
+             "output-virtualbox-iso/vbox-example.ovf"]
+  }
```

Comment:
These two examples are not entirely similar, is this okay?

Reply:
I think the json one also does it serially -- they're inside a set of double-brackets, which makes it a sequence definition, which means they'll run serially.

Reply:
Oh right ! I mistakenly thought that the first entry of the array was out of the array, sorry ! 👓